Mathlib/Data/Nat/Factorial/BigOperators.lean

```diff
@@ -23,6 +23,9 @@ open BigOperators Finset Nat
 
 namespace Nat
 
+lemma monotone_factorial : Monotone factorial := fun _ _ => factorial_le
+#align nat.monotone_factorial Nat.monotone_factorial
+
```

Comment:
There's already stuff about orders in there and I don't really feel like creating a new file just for this one lemma. I'm not quite sure where `Monotone` will stand in the upcoming sublibraries but I think it will be in `Order` and not imported by `Nat.factorial`.

I'm happy for you to open a PR creating that `Data.Nat.Factorial.Order` file if you want though, or maybe it can be in `Data.Nat.Order.Lemmas`. Up to you.